python/gink/impl/connection.py

```diff
@@ -27,12 +29,18 @@ def __init__(
             host: Optional[str] = None,
             port: Optional[int] = None,
             name: Optional[str] = None,
-            socket: Optional[Socket] = None,
-    ):
+            socket: Optional[Socket] = None
```

Comment:
The final argument of a multi-line arg list should have a trailing comma so that the addition of future arguments doesn't require changes in arguments that don't change.